spec/support/dashboard_helper.rb

```diff
@@ -1,38 +1,5 @@
-require_relative 'wait_for_ajax.rb'
-
 module DashboardHelper
-  extend ActiveSupport::Concern
-  include WaitForAjax
-
-  SHORT_DASHBOARD_REFRESH_SECONDS = 0.1
-
-  # Some integration tests run assertions dependant on refreshing the dashboard,
-  # this means there is an unavoidable sleep that needs to happen while we wait
-  # for the time interval to elapse to trigger a periodic refresh. If we need
-  # to do this, then we can temporarily set the refresh period to something
-  # shorter than standard.
-  #
-  # We only want to do this in tagged specs, reducing the refresh period for all
-  # tests will make things unstable if they are trying to interact with elements
-  # on the dashboard whilst it is refreshing.
-
-  included do
-    around(:example, short_dashboard_refresh: true) do |example|
-      old_refresh = Settings.dsds.dashboard_refresh_milliseconds
-      Settings.dsds.dashboard_refresh_milliseconds = short_dashboard_refresh_milliseconds
-      example.run
-      Settings.dsds.dashboard_refresh_milliseconds = old_refresh
-    end
-  end
-
-  def wait_for_dashboard_refresh
-    sleep(Settings.dsds.dashboard_refresh_milliseconds / 1000)
-    wait_for_ajax
-  end
-
-  private
-
-  def short_dashboard_refresh_milliseconds
-    SHORT_DASHBOARD_REFRESH_SECONDS * 1000
+  def refresh_dashboard
+    page.execute_script "window.jQuery.ajax({url: 'refresh_dashboard'})"
```

Comment:
I had a lot of doubts about this one. I ended up going this way for three reasons:

1) We were always "conditioning" the triggering by changing interval settings or other strategies;
2) This will expose failures in the polling mechanism in a much cleaner manner and make the test deterministic - all other variations seem to intermittently fail.
3) The entire test to me seems incorrect. We are manually updating records in the db at an integration level. In a way that is much more concerning than triggering the refresh at will;

Should we discuss other options?